charts/operator-wandb/templates/weave-trace.yaml

```diff
@@ -14,4 +14,32 @@ data:
   WANDB_BASE_URL: "http://{{ .Release.Name }}-app:8080/"
   {{- end }}
   WF_TRACE_SERVER_URL: "{{ .Values.global.host }}/traces"
-  WF_ENFORCE_PASSWORD_LENGTH: "false"
\ No newline at end of file
+  WF_ENFORCE_PASSWORD_LENGTH: "false"
+  WEAVE_ENABLE_ONLINE_EVAL: {{ (get  .Values "weave-trace-worker").install | quote }}
+  WEAVE_TRACE_SERVER_BASE_URL: "{{ .Values.global.host }}/traces"
+  WANDB_INTERNAL_SERVICE_TOKEN_SECRET_NAME: "weave-worker-auth"
+  KAFKA_BROKER_HOST: "bufstream.bufstream.svc.cluster.local"
```

Comment:
this should probably not be hard-coded in the chart.  The chart should use the values in global, and we should set this in the deployer channel.

Reply:
I could not find the value exposed in `global`. Please advise.

Reply:
Its not set by default because its a config specific to dedicated.  it should probably be set by deployer channel.  If what you have here works, we can circle back to refactor that later.